CONTRIBUTING.md

```diff
@@ -6,47 +6,62 @@ We love pull requests from everyone. By participating in this project, you agree
 to abide by the [code of conduct].
 
 [code of conduct]: CODE_OF_CONDUCT.md
+## How to contribute
+1. Fork the repo.
+2. Make your change.
+3. Test your change. Make sure all of the tests pass by running `./bin/rspec`
+4. Make sure your change matches our [slightly modified version] of the [Ruby Style Guide] style guide by running [rubocop]. `bundle exec rubocop`
+5. Write a [good commit message][commit].
+6. Include the github issue number(s) that this branch resolves. [More Info](https://help.github.com/articles/closing-issues-via-commit-messages/)
+7. Push to your fork.
+8. [Submit a pull request][pr].
+9. If [Hound] catches style violations, fix them.
 
-Fork the repo.
-
-Set up your machine:
-
-    ./bin/setup
+Wait for us.  We are also volunteers. We will get back to you as soon as possible.
+We may suggest changes.
 
-Make sure the tests pass:
+[slightly modified version]: https://github.com/PuzzledPint/puzzledpint.com/blob/master/.rubocop.yml
+[Ruby Style Guide]: https://github.com/bbatsov/ruby-style-guide
+[rubocop]: https://github.com/bbatsov/rubocop
+[commit]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
+[pr]: https://github.com/PuzzledPint/puzzledpint.com/compare/
+[Hound]: https://houndci.com
 
-    ./bin/rspec
 
-Make your change.
+## Setting up local development
+### Install dependencies
+1. Install postgresql: `brew install postgresql`
+2. [Follow these instructions](https://rvm.io) to install `rvm`.
+3. Install ruby 2.3.0. `rvm install ruby-2.3.0`
+4. Make a `tmp` directory. `mkdir tmp`
```

Comment:
Hmmmm...I am thinking something else with your set up...I'll ping you separately.